website/tfe.erb

```diff
@@ -0,0 +1,66 @@
+<% wrap_layout :inner do %>
+    <% content_for :sidebar do %>
+        <div class="docs-sidebar hidden-print affix-top" role="complementary">
+            <ul class="nav docs-sidenav">
+                <li<%= sidebar_current("docs-home") %>>
+                    <a href="/docs/providers/index.html">All Providers</a>
+                </li>
+
+                <li<%= sidebar_current("docs-tfe-index") %>>
+                    <a href="/docs/providers/tfe/index.html">Terraform Enterprise Provider</a>
+                </li>
+
+                <li<%= sidebar_current("docs-tfe-resource") %>>
+                    <a href="#">Resources</a>
+                    <ul class="nav nav-visible">
+                        <li<%= sidebar_current("docs-tfe-organization") %>>
+                            <a href="/docs/providers/tfe/r/organization.html">tfe_organization</a>
+                        </li>
+
+                        <li<%= sidebar_current("docs-tfe-organization-token") %>>
+                            <a href="/docs/providers/tfe/r/organization_token.html">tfe_organization_token</a>
+                        </li>
+
+                        <li<%= sidebar_current("docs-tfe-sentinel-policy") %>>
+                            <a href="/docs/providers/tfe/r/sentinel_policy.html">tfe_sentinel_policy</a>
+                        </li>
+
+                        <li<%= sidebar_current("docs-tfe-ssh-key") %>>
+                            <a href="/docs/providers/tfe/r/ssh_key.html">tfe_ssh_key</a>
+                        </li>
+
+                        <li<%= sidebar_current("docs-tfe-team") %>>
```

Comment:
Same here -- this `sidebar_current` will highlight all the other team ones unless its updated to not be a prefix of the others.

Reply:
Fixed as well...